frontend/src/views/SecretMainPage/components/SecretImportListView/SecretImportListView.tsx

```diff
@@ -159,7 +159,15 @@ export const SecretImportListView = ({
       >
         <SortableContext items={items} strategy={verticalListSortingStrategy}>
           {items?.map((item) => {
-            const { importPath, importEnv, id, isReplication } = item;
+            const {
+              importPath,
+              importEnv,
+              id,
+              isReplication,
+              replicationStatus,
+              lastReplicated,
+              isReplicationSuccess
+            } = item;
```

Comment:
The replicated secret is not deleted after the import is unlinked. is this intentional? Icon still seem to indicate that it's a replicated secret so might be misleading
<img width="661" alt="image" src="https://github.com/Infisical/infisical/assets/65645666/b51652bb-207a-4d3c-acad-cfeaa0e3dc78">

Reply:
Its expected behavior.